### PR TITLE
Status field added and exposed on UI

### PR DESCRIPTION
**Pull request by Roboneet**
In addition to the above mentioned change , read_dotenv in celery.py was changed to avoid a warning (screen shot attached)
screen shot 2017-04-07 at 6 26 51 pm

### CODE REVIEW

**Comment by comzeradd** [comment]
Hi @Roboneet. Thanks for working on this. 

I think you misread the issue and the specs. The proposed status options refer to the Satellites, not the transmitters.

**Comment by Roboneet** [comment]
Oh , my bad!  I'll fix that

File: db/base/models.py [hunk 1]
```diff
@@ -58,6 +58,19 @@ class Satellite(models.Model):
...
     tle2 = models.CharField(max_length=200, blank=True)
+    ALIVE = 'a'
+    DEAD = 'd'
+    RE_ENTERED = 'r'
+    STATUS = (
+        (ALIVE, 'Alive'),
+        (DEAD, 'Dead'),
+        (RE_ENTERED, 'Re-entered'),
+    )
+    status = models.CharField(
```
Comment (comzeradd) [inline db/base/models.py]: I suggest you use the same code convetions with the rest of the code. See the `source` field on the `DemodData` model and the `DATA_SOURCES` variable. So place the `STATUS` variable on top (maybe name it something like `SATELLITE_STATUS` to make more sense) and try to keep the model code as simple as possible.

**Comment by Roboneet** [comment]
The tests work fine on my system.
screen shot 2017-04-17 at 10 32 48 am

File: db/base/models.py [hunk 2]
```diff
@@ -11,6 +11,8 @@
 from db.base.helpers import gridsquare
 
 DATA_SOURCES = ['manual', 'network', 'sids']
+SATELLITE_STATUS = ['Alive', 'Dead', 'Re-entered']
+SATELLITE_STATUS_CODES = ['a', 'd', 'r']
```
Comment (comzeradd) [inline db/base/models.py]: Why use two separete dicts? If you want to have a short code to use on template later for css you can do that with one dict and don't use `zip` on the model. See [this example](https://docs.djangoproject.com/en/1.11/ref/models/fields/#choices) from the documentation.

**Comment by comzeradd** [comment]
Hey @Roboneet. Did you had the chance to see the requested changes?

**Comment by comzeradd** [comment]
Many thanks @Roboneet 👍 That looks great.

I opened a new PR at #173 to rebase your commits and simplify the model a bit. Thanks again.

**Pull request closed without merging**